controllers/flp/flp_pipeline_builder.go

```diff
@@ -79,8 +81,18 @@ func (b *PipelineBuilder) AddProcessorStages() error {
 					"SrcAddr",
 					"DstAddr",
 				},
-				Output:      "K8S_FlowLayer",
-				InfraPrefix: b.desired.Namespace,
+				Output:        "K8S_FlowLayer",
+				InfraPrefixes: []string{b.desired.Namespace, openshiftNamespacesPrefixes},
+				InfraRefs: []api.K8sReference{
+					{
+						Name:      "kubernetes",
+						Namespace: "default",
+					},
+					{
+						Name:      "openshift",
+						Namespace: "default",
+					},
```

Comment:
what's this exactly, do you know? I've never seen it used / never appears in my flows

Reply:
Two services are deployed in the default namespace, `kubernetes` and `openshift` they can be used to reach respectively kubernetes and openshift API.

Now, I don't exactly know what is exactly exposed in the openshift API.